transcom/mymove
# B 21169 int pr3

## Summary

This PR moves the checking of the Alaska and Hawaii feature flags to the Handler level instead of the Service level. The service level seems to have issues with getting the correct status of the Feature Flag. I also temporarily added some logging to check to make sure the Boat and Mobile Home Feature Flags are being read properly since they are also checked at the Service level. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

For this test we need to make sure the FF is being read properly for AK and HI. 

1. Make sure the AK FF is on 
2. Login as an Office user or Customer and do an address search for an AK city or zip and make sure you see the results
3. Turn the AK FF off and verify you don't see any AK results
4. Repeat the above with the HI FF

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
